buildenv/jenkins/jobs/pull-requests/PullRequest-Compile-JDK10-aix_ppc-64_cmprssptrs

```diff
@@ -0,0 +1,39 @@
+/*******************************************************************************
+ * Copyright (c) 2018, 2018 IBM Corp. and others
+ *
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+ * or the Apache License, Version 2.0 which accompanies this distribution and
+ * is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * This Source Code may also be made available under the following
+ * Secondary Licenses when the conditions for such availability set
+ * forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+ * General Public License, version 2 with the GNU Classpath
+ * Exception [1] and GNU General Public License, version 2 with the
+ * OpenJDK Assembly Exception [2].
+ *
+ * [1] https://www.gnu.org/software/classpath/license.html
+ * [2] http://openjdk.java.net/legal/assembly-exception.html
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+ *******************************************************************************/
+
+SDK_VERSION = '10'
+SPEC = 'aix_ppc-64_cmprssptrs'
+
+node('master') {
+
+    checkout scm
+    variableFile = load 'buildenv/jenkins/common/variables-functions'
+    variableFile.set_job_variables('build')
+    cleanWs()
+}
+
+node("${NODE}") {
+
+    checkout scm
+    buildfile = load 'buildenv/jenkins/common/build'
+    buildfile.build_pr_compile_only()
```

Comment:
In other scripts is: `buildfile.build_all()` instead `buildfile.build_pr_compile_only()`